samples/ORKCatalog/ORKCatalog/TaskListRow.swift

```diff
@@ -289,6 +294,9 @@ enum TaskListRow: Int, Printable {
             
             case .Form:
                 return formTask
+
+            case .ToneAudiometry:
```

Comment:
Since this is sample code, it's helpful to retain the same order everywhere. If you feel the right place is after audio, put this case just below audio.